### DISCUSSION TITLE
Contributing to docs

### DISCUSSION THREAD

**Discussion by rudolfbyker** [TanStack/query#6885] [Q&A]
I have read CONTRIBUTING.md and README.md and most of the online docs, but I can't find a way to build the docs locally so that I can contribute to the docs. What framework is this using? Is there a script somewhere to preview the docs?

**Comment by TkDodo** [comment]
docs are in the tanstack.com repo:

https://github.com/TanStack/tanstack.com

I just write markdown and wait for deployment :)

**Reply by rudolfbyker** [reply]
Hi @TkDodo . Thanks for taking the time to answer. However, this does not answer my question at all. I already know where the docs are. I want to know how to preview / build them locally.

**Comment by TkDodo** [comment]
https://github.com/TanStack/tanstack.com/blob/main/README.md
